spec/features/reviews_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "rails_helper"
+require "pry"
+​
+  feature "Create A Review" do
+    scenario "logged in user successfully creates a new review" do
+
+      new_user = User.create(email: "mckelvey.matt@gmail.com", password: "12345678")
+    ​
+      visit user_session_path
+    ​
+      fill_in 'Email', with: 'mckelvey.matt@gmail.com'
+      fill_in 'Password', with: '12345678'
+    ​
+      click_button "Log in"
+    ​
+      click_button "Add New Bar"
+    ​
+      fill_in 'Name', with: 'Beantown Pub'
+      fill_in 'Description', with: 'After hours cocktail bar with pool.'
+    ​
+      click_button "Add Bar"
+
+      click_button "Add Review"
+      fill_in 'title', with: "Best bar I've been to"
+      fill_in 'body', with: "We were visiting and had a good time"
+      fill_in 'rating', with: 5
+      click_button "Create My Review"
+      expect(page).to have_content("You've succesfully added a review")
+      expect(page).to have_content("Best bar I've been to")
+
+    end
+
+      scenario "non-logged in user cannot create a new review" do
+
+      bar = Bar.new(name: 'A', :description: 'B')
+
+      visit bars_path
+      click_link 'A'
+      click_button 'Add Review'
+      expect(page).to have_content('You must sign in first')
+      expect(page).to have_content('Login')
+      expect(page).to have_content('Email')
+    end
+  end
+
+  feature "Edit Reviews" do
```

Comment:
Each of these `feature` blocks should be in a separate, descriptively-named file!